hw03.py

```diff
@@ -0,0 +1,33 @@
+x = input("Write something:")
+if x.isdigit():
+    if int(x) % 2 == 0:
+        print("This is a number")
+        print("Number is even")
+
+    elif int(x) % 2 == 1:
+            print("This is a number")
+            print("Number is odd")
+
+elif x.isalpha():
+    if x.islower():
+        print("This is a letter")
+        print("This is uppercase letter")
+    elif x.isupper():
```

Comment:
You can simply use `else` here since if a character is not lowercase it is uppercase and there is no need to additionally check it